02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,96 @@
+const express = require("express");
+const router = express.Router();
+const {
+  addPerson,
+  getPeople,
+  getPerson,
+  updatePerson,
+  removePerson
+} = require("../controllers/people");
+
+router.get("/", (request, response) => {
```

Comment:
All of your controller logic should be in your imported controller function.  For `router.get()`, the call should look like:
```javascript
router.get('/', getPerson)
```

Reply:
@tryandbry, Controller logics separated from routes, also fake database logics moved to services.js in 3fec47a 
Thanks